### PR TITLE
driver.rkt: Remove unnecessary job validation

### PR DESCRIPTION
**Pull request by nak3**

### CODE REVIEW

**Comment by nak3** [comment]
`DefaultResources()` and `Validate()` checked them. And other drivers don't have this validation, but only rkt driver has this.

**Comment by dadgar** [comment]
Thanks!

File: client/driver/rkt.go [hunk 1]
```diff
@@ -175,11 +175,9 @@ func (d *RktDriver) Start(ctx *ExecContext, task *structs.Task) (DriverHandle, e
 	if err := mapstructure.WeakDecode(task.Config, &driverConfig); err != nil {
 		return nil, err
 	}
-	// Validate that the config is valid.
+
+	// ACI image
 	img := driverConfig.ImageName
-	if img == "" {
```
Comment (diptanu) [inline client/driver/rkt.go]: @nak3 Why was this removed?
